sysinit/coqinit.ml

```diff
@@ -10,6 +10,42 @@
 
 (** GC tweaking *)
 
+(** Work around a broken Gc.set function in released OCaml versions between
+    4.08.0 and 4.11.1 inclusive. *)
+let gc_set : Gc.control -> unit =
+  let open Gc in
+  let fixed_set control =
+    let { custom_major_ratio; custom_minor_ratio; custom_minor_max_size; _ } = control in
+    let control =
+      {
+        control with
+        custom_major_ratio = custom_major_ratio lsr 1;
+        custom_minor_ratio = custom_minor_ratio lsr 1;
+        custom_minor_max_size = custom_minor_max_size lsr 1;
+      }
+    in
+    Gc.set control
+  in
+  let v =
```

Comment:
you might want to use `Coq_config.caml_version_nums` like in `init_gc`

Reply:
Right, thanks.